docs/tutorial-using-verify-service-provider.md

```diff
@@ -18,7 +18,6 @@ This tutorial explains how to:
 To be able to follow this tutorial you must:
 * have Java 8
```

Comment:
Isn't everything on Java 11 now?

Reply:
I would have thought have at least Java 8 would more appropriate since this can work on Java 11 as well.